package.json

```diff
@@ -14,7 +14,7 @@
     "deploy": "sh scripts/deploy.sh"
   },
   "dependencies": {
-    "@artsy/reaction-force": "^0.1.49",
+    "@artsy/reaction-force": "^0.3.1",
```

Comment:
I'm going to go ahead and deploy reaction -- is still on 3.1, do i jump right to v4.0?